source/_integrations/matter.markdown

```diff
@@ -149,7 +149,7 @@ This will use the Bluetooth connection of your phone to add the device.
 
 ### To add a new device using the Android Companion app
 
-This will use the Bluetooth connection of your phone to add the device.
+This guide describes how to add a new device that has never been added to your network before. This will use the Bluetooth connection of your phone to add the device.
```

Comment:
is this clear enough ? Or do you want to refer to "just unpacked from the box" ?

Reply:
Well, the device I used wasn't really new new. It was just no longer paired to any server and freshly reset.